src/server.c

```diff
@@ -3233,6 +3232,31 @@ void propagatePendingCommands() {
     redisOpArrayFree(&server.also_propagate);
 }
 
+/* Performs operations that should be performed after an execution unit ends.
+ * Execution unit is a code that should be done atomically.
+ * Logical units can be nested and are not necessarily starts with Redis command.
```

Comment:
ohh, now i see where "Unit" comes from.
if we're coining the term "Execution Unit" for this thing, i think we may need to mention it in a few other places, not sure where.
but at the very least, let's rename the `postUnitOperations` to `postExecutionUnitOperations`.
The term "unit" can be applied to so many other things, and i think "exec" as short hand for "execution" isn't good since we have a command named EXEC.

Reply:
So I guess we should also rename `AddPostJob` to `AddPostExecutionUnitJob`

Reply:
Updated, let me know if it makes more sense now.